specification/iotoperations/IoTOperations.Management/models/akri/common.tsp

```diff
@@ -0,0 +1,219 @@
+@added(Versions.`2025-07-01-preview`)
+model AkriConnectorsServiceAccountAuthentication
+  extends AkriConnectorsMqttAuthentication {
+  /** The authentication method for the MQTT connection. */
+  method: ServiceAccountToken;
+
+  /** The service account token for the MQTT connection. */
+  serviceAccountTokenSettings: AkriConnectorsServiceAccountTokenSettings;
+}
+
+/** AkriConnectorsServiceAccountTokenSettings properties. */
+@added(Versions.`2025-07-01-preview`)
+model AkriConnectorsServiceAccountTokenSettings {
+  /** The audience for the service account token. */
+  audience: string;
+}
+
+/** Image pull policy. */
+@added(Versions.`2025-07-01-preview`)
+union AkriConnectorsImagePullPolicy {
+  string,
+
+  /** Always pull the image. */
+  Always,
+
+  /** IfNotPresent pull the image. */
+  IfNotPresent,
+
+  /** Never pull the image. */
+  Never,
+}
+
+/** AkriConnectorsImagePullSecret properties. */
+@added(Versions.`2025-07-01-preview`)
+model AkriConnectorsImagePullSecret {
+  /** The name of the image pull secret. */
+  secretRef: string;
+}
+
+/** AkriConnectorsTagDigestSettings properties. */
+@added(Versions.`2025-07-01-preview`)
+@discriminator("tagDigestType")
+model AkriConnectorsTagDigestSettings {
+  /** The tag or digest type. */
+  tagDigestType: AkriConnectorsTagDigestType;
+}
+/** AkriConnectorsTagDigestType values. */
+@added(Versions.`2025-07-01-preview`)
+union AkriConnectorsTagDigestType {
+  string,
+
+  /** Indicates that a tag should be specified. */
+  Tag,
+
+  /** Indicates that a digest should be specified. */
+  Digest,
+}
+
+/** AkriConnectorsTag properties. */
+@added(Versions.`2025-07-01-preview`)
+model AkriConnectorsTag extends AkriConnectorsTagDigestSettings {
+  /** The tag or digest type. */
+  tagDigestType: Tag;
+
+  /** The tag of the image. */
+  tag: string;
+}
+
+/** AkriConnectorsDigest properties. */
+@added(Versions.`2025-07-01-preview`)
+model AkriConnectorsDigest extends AkriConnectorsTagDigestSettings {
+  /** The tag or digest type. */
+  tagDigestType: Digest;
+
+  /** The digest of the image. */
+  digest: string;
+}
+/** AkriConnectorsRegistrySettings properties. */
+@added(Versions.`2025-07-01-preview`)
+@discriminator("registrySettingsType")
+model AkriConnectorsRegistrySettings {
+  registrySettingsType: AkriConnectorsRegistrySettingsType;
+}
+
+/** AkriConnectorsRegistrySettings properties. */
+@added(Versions.`2025-07-01-preview`)
+union AkriConnectorsRegistrySettingsType {
+  string,
+
+  /** A Registry Endpoint reference. */
+  RegistryEndpointRef,
+
+  /** A Container Registry reference. */
+  ContainerRegistry,
+}
+
+/** AkriConnectorsRegistryEndpointRef properties. */
+@added(Versions.`2025-07-01-preview`)
+model AkriConnectorsRegistryEndpointRef extends AkriConnectorsRegistrySettings {
+  /** The registry endpoint reference. */
+  registrySettingsType: RegistryEndpointRef;
+
+  /** The name of the registry endpoint. */
+  registryEndpointRef: string;
```

Comment:
I wonder why call it a registeryEndpointRef and not just registryEndpoint?  Why is it a 'name of'? In what sense is the name going to be dereferenced?